bootcamp/RAG/RAG_Milvus_LangChain_Anthropic.ipynb

```diff
@@ -0,0 +1,1168 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.                connections.connect("default", host=self.host, port=self.port)
</pre><p>This is deprecated API. Recommend to use MilvusClient(). Also it's suggested to use endpoint that composes the host and port for initializing MilvusClient()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1565/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>